cpp/ql/lib/semmle/code/cpp/File.qll

```diff
@@ -308,13 +288,6 @@ class File extends Container, @file {
    */
   override predicate fromSource() { numlines(underlyingElement(this), _, _, _) }
 
-  /**
-   * Holds if this file may be from a library.
-   *
-   * DEPRECATED: For historical reasons this is true for any file.
-   */
-  deprecated override predicate fromLibrary() { any() }
```

Comment:
Oooh, I think this one's existed in a deprecated state since around the time I started at Semmle.  About time!